Modules/OptionHolder.cs

```diff
@@ -2670,6 +2677,11 @@ public static void Load()
             .SetValueFormat(OptionFormat.Times);
         AutoKickStartAsBan = BooleanOptionItem.Create(19312, "AutoKickStartAsBan", false, TabGroup.SystemSettings, false)
             .SetParent(AutoKickStart);
+
+        TempBanPlayersWhoKeepQuitting = BooleanOptionItem.Create(44440, "TempBanPlayersWhoKeepQuitting", false, TabGroup.SystemSettings, false);
+        QuitTimesTillTempBan = IntegerOptionItem.Create(44441, "QuitTimesTillTempBan", new(1, 15, 1), 4, TabGroup.SystemSettings, false)
+            .SetParent(TempBanPlayersWhoKeepQuitting)
```

Comment:
And it's unnecessary

Reply:
Yeah delete it

Reply:
My bad, I thought there was only 1 setting